releasenotes/notes/apm-automaxprocs-41ca2da9b16c36e3.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: The trace-agent now automatically sets the GOMAXPROCS value in
+    Linux containers to whatever the container has allocated, as opposed
```

Comment:
Nit: I think "to match Linux CPU quota" would be more precise than "whatever the container has allocated".

Reply:
👍 I improved it a bit. PTAL